src/accounts/AccountTabs.tsx

```diff
@@ -1,66 +1,74 @@
 // Libraries
 import React, {FC} from 'react'
-import {useSelector} from 'react-redux'
 import {Link} from 'react-router-dom'
+import {useSelector} from 'react-redux'
 
 // Components
-import {Tabs, Orientation, ComponentSize} from '@influxdata/clockface'
+import {ComponentSize, Orientation, Tabs} from '@influxdata/clockface'
 
 // Utils
 import {getOrg} from 'src/organizations/selectors'
+import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 
 interface Props {
   activeTab: string
 }
 
 interface AccountPageTab {
-  text: string
   id: string
+  enabled: boolean
   link: string
   testID: string
-}
-
-enum Tab {
-  Billing = 'billing',
-  About = 'about',
+  text: string
 }
```

Comment:
Added `enabled` property to this array so that our render function knows which tabs to display and which not to display, based on whether the `createDeleteOrgs` flag is on.